config/wdio/visualRegressionConf.js

```diff
@@ -70,10 +70,10 @@ function getScreenshotPath(ref) {
 
     const baseDir = global.browser.options.baseScreenshotDir;
     if (baseDir) {
-      testPath = testPath.split(process.cwd())[1];
+      [, testPath] = testPath.split(process.cwd());
```

Comment:
Linter wanted this :)

Reply:
yeah, I had disabled this eslint rule previously since we do so much with objects in the repo. https://github.com/cerner/terra-toolkit/pull/253/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9L13